libcxx/test/std/containers/sequences/array/empty.verify.cpp

```diff
@@ -6,17 +6,17 @@
 //
 //===----------------------------------------------------------------------===//
 
+// REQUIRES: std-at-least-c++20
+
```

Comment:
I think we can remove these lines now, although this is a bit beyond NFC.
```suggestion
```

Reply:
This is  a test about `[[nodiscard]]` actually. It's redundant. I removed it.